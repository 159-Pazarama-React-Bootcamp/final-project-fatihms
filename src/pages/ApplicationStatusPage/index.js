import React from "react";

import Header from "../../components/Header";
import ApplicationStatusForm from "../../containers/ApplicationStatusForm";

import styles from "./styles.module.css";

function ApplicationStatusPage() {
  return (
    <body className={styles["application-status-container"]}>
      <header>
        <Header />
      </header>
      <main>
        <article>
          <ApplicationStatusForm />
        </article>
      </main>
    </body>
  );
}

export default ApplicationStatusPage;
