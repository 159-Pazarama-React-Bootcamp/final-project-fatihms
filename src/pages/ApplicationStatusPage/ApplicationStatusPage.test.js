/*eslint-disable */

import React from "react";

import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import { MemoryRouter as Router } from "react-router-dom";
import userEvent from "@testing-library/user-event";
import "@testing-library/jest-dom/extend-expect";

import ApplicationStatusPage from "./index";

describe("ApplicationStatusPage", () => {
  beforeEach(() => {
    render(
      <Router>
        <ApplicationStatusPage />
      </Router>
    );
  });
  test("ApplicationStatusPage renders correctly", () => {
    const tree = renderer
      .create(
        <Router>
          <ApplicationStatusPage />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
