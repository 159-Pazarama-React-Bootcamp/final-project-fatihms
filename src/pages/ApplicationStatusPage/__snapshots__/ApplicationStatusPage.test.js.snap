// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationStatusPage ApplicationStatusPage renders correctly 1`] = `
<div
  className="application-status-container"
>
  <header>
    <div
      className="header-container"
    >
      <h2>
        Ba≈üvuru Sorgula
      </h2>
    </div>
  </header>
  <main>
    <article>
      <section
        className="application-status-page"
      >
        <div
          className="form-container form-container--20"
        >
          <form
            onSubmit={[Function]}
          >
            <div
              className="input-container"
            >
              <input
                id="code"
                name="code"
                onChange={[Function]}
                placeholder="g2G'T)W'S"
                type="text"
                value=""
              />
              <label
                htmlFor="code"
              >
                Kod
              </label>
            </div>
            <button
              className="button-item"
              type="submit"
            >
              Sorgula
            </button>
          </form>
        </div>
      </section>
    </article>
  </main>
</div>
`;
